src/main/java/main/java/Demo.java

```diff
@@ -0,0 +1,26 @@
+package main.java;
+
+public class Demo {
+    public static void main(String[] args) {
+        int[] moduleMass = {50200, 50204, 50250, 51639, 51898, 52873, 53057, 53595, 53965, 56280, 61024, 65123, 66332, 67987, 70417, 72688, 73140, 73658, 74299, 74819, 75185,
+                75468, 75662, 77151, 77417, 80054, 80924, 81075, 84734, 85822, 86208, 86881, 88562, 88811, 88877, 89008, 89504, 89875, 90128, 91624, 91978,
+                93188, 93398, 94047, 94330, 94512, 98199, 98383, 98851, 99960, 101008, 102818, 103326, 103934, 104476, 106523, 107136, 110176, 110233,
+                111192, 112376, 113345, 113805, 114558, 114748, 115146, 115571, 117845, 118483, 118711, 120203, 121281, 122374, 124360, 124950, 125638, 125892,
+                127929, 127943, 128132, 130211, 131326, 135418, 136289, 137867, 138366, 138631, 138909, 140186, 140638, 140787, 141095, 142697, 143477, 143538, 144819, 146183, 147701, 147914, 149923};
+
+        System.out.println(fuel(moduleMass));
+
+
+    }
+
+    public static int fuel(int[] list) {
```

Comment:
Function names are verbs, not nouns. `CalculateTotalRocketFuel` (could be shortened probably) though verbose, is much more readable